src/main/java/Daisy.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+public class Daisy {
+    public static void main(String[] args) {
+        final String INTRO_PROMPT = "Good day! This is Daisy.\nAny task for today?";
+        final String EXIT_PROMPT = "Ending prompt received. Remember to keep to the deadlines!";
+        final String LINE_BREAK = "____________________________________";
+        Task[] tasks = new Task[100];
```

Comment:
Perhap this magic number can be avoided? Using a named constant (Like you did for your literal above) will allow others to understand the meaning of this number.